src/communities/index.ts

```diff
@@ -19,7 +19,7 @@ const communities = (
 ): IRestResource<ICommunity | IMHBOListing> => ({
   byIds: (params: number[]) =>
     requestGet<ICommunity, IUnparsedCommunity>(
-      `v1/communities/?${params.toString()}&detail_level=FULL`,
+      `v1/communities/${params.toString()}&detail_level=FULL`,
```

Comment:
The first parameter still needs the "?". This is only working because it is defaulting to FULL.
v1/communities/${params.toString()}?detail_level=FULL

Reply:
resolved